src/components/universal/core/Badge/Badge.recipe.ts

```diff
@@ -0,0 +1,42 @@
+import { defineRecipe } from "@pandacss/dev";
+
+export const badgeRecipe = defineRecipe({
+  className: "badge",
+  description: "The styles for the Badge component",
+  base: {
+    borderRadius: "full",
+    fontWeight: "medium",
+    display: "inline-flex",
+    alignItems: "center",
+    userSelect: "none",
+    whiteSpace: "nowrap",
+  },
+  defaultVariants: {
+    variant: "subtle",
+    size: "md",
+  },
+  variants: {
+    variant: {
+      solid: {
+        background: "accent.default",
+        color: "accent.fg",
+      },
+      subtle: {
+        background: "bg.subtle",
+        borderColor: "border.default",
+        borderWidth: "1px",
+        color: "fg.emphasized",
+      },
+      outline: {
+        borderWidth: "2px",
+        borderColor: "border.emphasized",
+        color: "fg.emphasized",
+      },
+    },
+    size: {
+      sm: { textStyle: "xs", px: "2", h: "5", gap: "1" },
```

Comment:
Use numeric values when applicable (`px: 2`, etc)

Reply:
I remember this when looking at panda docs in the past [https://panda-css.com/docs/concepts/writing-styles#numeric-values](url). Is this anything to be concerned about in the future?

Reply:
Ahhh yeah so this is about auto converting to pixels. So like, for example, with tailwind if you want some middle of the road value you would do something like `mx-[13.25px]` or whatever. In this case, when you are just using numeric values that are predefined in panda, you will be good!